### PR TITLE
674 - speciality blog background color

### PR DESCRIPTION
**Pull request by omonk**
## Description - [Trello ticket](https://trello.com/c/ROW7kGFV/674-speciality-pages-visual-discrepancies-audit)

Update query to include new `blogBackgroundColor` field, adds conditional wrapper around blog and events sections in speciality

### CODE REVIEW

File: src/components/Common/BlogListing.js [hunk 1]
```diff
@@ -5,8 +5,22 @@ import { format } from 'date-fns'
...
+      return <GreyBackground>{children}</GreyBackground>
+    default:
+      return <>{children}</>
+  }
+}
+const BlogListing = ({
+  title,
+  description,
+  posts,
+  blogBackgroundColor = 'White'
```
Comment (fabiommmoreira) [inline src/components/Common/BlogListing.js]: should be lowercase for consistency

Comment (fabiommmoreira) [inline src/components/Common/BlogListing.js]:
```suggestion
  blogBackgroundColor = 'white'
```

File: src/components/Common/BlogListing.js [hunk 2]
```diff
@@ -15,18 +29,20 @@ const BlogListing = ({ title, description, posts }) => {
...
-          mediaItems={mediaItems}
-          CTALink="/blog/"
-          external
-          CTAText="Visit our blog"
-        />
-      </Padding>
-    </Grid>
+    <Wrapper bgColor={blogBackgroundColor}>
+      <Grid>
+        <Padding vertical={{ desktop: 5, smallPhone: 3.5 }}>
```
Comment (fabiommmoreira) [inline src/components/Common/BlogListing.js]: Can we put the Padding component above the Grid? That way we could keep the Grid > Row > Col scheme

Comment (omonk) [reply src/components/Common/BlogListing.js]: Yeah, this is annoying as the TitleAndMediaList component has `RowLayout` internally. I would opt for removing it altogether but not sure where to put the padding instead as I don't really feel it should be on the grid

**Pull request merged**